project-base/src/Shopsys/ShopBundle/DataFixtures/Demo/TransportDataFixture.php

```diff
@@ -98,6 +125,20 @@ protected function createTransport($referenceName, TransportData $transportData)
         $this->addReference($referenceName, $transport);
     }
 
+    /**
+     * @param \Shopsys\ShopBundle\Model\Transport\TransportData $transportData
+     * @param \Shopsys\FrameworkBundle\Component\Money\Money $price
+     */
+    protected function setPriceForAllCurrencies(TransportData $transportData, Money $price): void
+    {
+        foreach ($this->domain->getAllIncludingDomainConfigsWithoutDataCreated() as $domain) {
+            $currency = $this->currencyFacade->getDomainDefaultCurrencyByDomainId($domain->getId());
+            $price = $this->priceConverter->convertPriceWithoutVatToPriceInDomainDefaultCurrency($price, $domain->getId());
+
+            $transportData->pricesByCurrencyId[$currency->getId()] = $price;
+        }
+    }
+
```

Comment:
same as in `PaymentDataFixture`
Just a thought.. Why don't you just get all currencies and set it for them instead of getting them from domains?

Reply:
because I just need the default currencies, renamed in f8eedbbb39e93d23fbc6ff2eb8800363f8e0124f